types/node/index.d.ts

```diff
@@ -92,6 +92,7 @@
 /// <reference path="dns/promises.d.ts" />
 /// <reference path="dns/promises.d.ts" />
 /// <reference path="domain.d.ts" />
+/// <reference path="dom-events.d.ts" />
```

Comment:
I'm surprised this passes tests given that `dom-events` should be parsed as a module.

Reply:
Ah, well, I may be showing a lot of ignorance in this PR but at least I'm learning a lot too.

Is the opposite of "parsed as a module" supposed to be "parsed as a script"?  And all the other `reference`d files parse as a script because at the top level they only `declare module 'xyz'`?  (Or in the case of `globals.d.ts`, declare types that will be added to the global scope.)

Should I refactor the DOM event declarations to parse as a script?  (Is that possible without exporting all the intermediate types?)  If not, would it be more idiomatically correct to use a side-effect `import` statement instead?  I was just aping the syntax used to incorporate the other files, not realizing how they were different.